packages/browser/src/Component/Widget/Server/SPARQL/View/SparqlQueryWindow.js

```diff
@@ -19,8 +21,16 @@ export class SparqlQueryWindow extends Window {
    * @param {SparqlEndpointResponseProvider} sparqlProvider The SPARQL Endpoint Response Provider
    * @param {CityObjectProvider} cityObjectProvider The City Object Provider
    * @param {LayerManager} layerManager The UD-Viz LayerManager.
+   * @param {object} configSparql The sparqlModule configuration.
+   * @param {object} configSparql.queries Query configurations
```

Comment:
config of assets/config/widget/sparql.json

Reply:
added to assets/config/widget/sparql_widget.json